gradle/tests/test.bats

```diff
@@ -0,0 +1,27 @@
+source "${BATS_TEST_DIRNAME}/../plan.sh"
+
+@test "Version matches" {
+  result="$(gradle --version | head -n 3 | tail -n 1 | awk '{print $2}' | tr -d ' ')"
+  [ "$result" = "${pkg_version}" ]
+}
+
+
+@test "Run A Scan With Gradle" {
+  rm -rf "/hab/cache/src/hello-gradle"
+  mkdir -p "/hab/cache/src/hello-gradle"
+  cd "/hab/cache/src/hello-gradle"
+
+  echo "buildScan {" >> "/hab/cache/src/hello-gradle/build.gradle"
```

Comment:
These are probably better suited to being placed in a test file, to clean up the `bats` file, and make diffs on testing easier to parse later.